### ISSUE TITLE
BuckleScript 7.2.2 version problem

### ISSUE THREAD

**Issue by erlandsona** [ELLIOTTCABLE/bs-deriving#6]
```
internal/modules/cjs/loader.js:613
    throw err;
    ^

Error: Cannot find module 'bs-platform/lib/bsc.exe'
Require stack:
- /Users/aerlandson/code/aspire-client/node_modules/ppx-deriving/identify.js
- /Users/aerlandson/code/aspire-client/node_modules/ppx-deriving/ppx.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:610:15)
    at Function.resolve (internal/modules/cjs/helpers.js:21:19)
    at Object.<anonymous> (/Users/aerlandson/code/aspire-client/node_modules/ppx-deriving/identify.js:11:30)
    at Module._compile (internal/modules/cjs/loader.js:759:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:770:10)
    at Module.load (internal/modules/cjs/loader.js:628:32)
    at Function.Module._load (internal/modules/cjs/loader.js:555:12)
    at Module.require (internal/modules/cjs/loader.js:666:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/Users/aerlandson/code/aspire-client/node_modules/ppx-deriving/ppx.js:22:22)

  We've found a bug for you!
  /Users/aerlandson/code/aspire-client/src/Store.re

  Error while running external preprocessor
Command line: /Users/aerlandson/code/aspire-client/node_modules/ppx-deriving/ppx.js '/var/folders/3h/fwbz7d9n3rx3v9q3dpm0rxt8f1vjgw/T/camlppxab103b' '/var/folders/3h/fwbz7d9n3rx3v9q3dpm0rxt8f1vjgw/T/camlppxf81b3f'
```

**Comment by hejfelix** [comment]
Same!

**Comment by hejfelix** [comment]
Can confirm:

- 7.2.2 fails
- 7.2.1 fails
- 7.3.1 fails
- 7.1.1 works!

**Comment by Coobaha** [comment]
https://github.com/ELLIOTTCABLE/ppx-deriving/pull/3

**Comment by ELLIOTTCABLE** [comment]
Merged and published in [v44.1.2](/ELLIOTTCABLE/bs-deriving/releases/tag/v44.1.2). (=

I can't currently test this thoroughly, so do let me know if you have any issues with the solution; but it looks like it should be fine until the next BuckleScript break.
